src/osd/PeeringState.cc

```diff
@@ -6666,7 +6666,7 @@ void PeeringState::ToDelete::exit()
 PeeringState::WaitDeleteReserved::WaitDeleteReserved(my_context ctx)
   : my_base(ctx),
     NamedState(context< PeeringMachine >().state_history,
-	       "Started/ToDelete/WaitDeleteReseved")
+	       "Started/ToDelete/WaitDeleteReserved")
```

Comment:
I'm inclined to accept, @epuertat  ?

Reply:
Yeah, that's only visible in logs (grep'ed through the code base and Googled that).